connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorMakerConfig.java

```diff
@@ -175,6 +175,7 @@ public MirrorClientConfig clientConfig(String cluster) {
         props.putAll(stringsWithPrefix("header.converter"));
         props.putAll(stringsWithPrefix("task"));
         props.putAll(stringsWithPrefix("worker"));
+        props.putAll(stringsWithPrefix("replication.policy"));
```

Comment:
In MirrorClientConfig ,it seems that it's not necessary to add replication.policy  into the props.
As i see , org.apache.kafka.connect.mirror.MirrorClientConfig#replicationPolicy  initialise the ReplicationPolicy  instance,
and  in  MirrorClient this instance is used ,where ReplicationPolicy   takes effect for real.